aptos-move/e2e-testsuite/src/tests/validator_set_management.rs

```diff
@@ -319,17 +319,13 @@ fn validator_set_operator_set_key_reconfigure() {
         executor.new_block();
 
         // DR sets operator 1 for validator 0
-        let admin_script = encode_set_validator_operator_with_nonce_admin_script(
+        let admin_script_payload = encode_set_validator_operator_with_nonce_admin_script_function(
             0,
             b"operator_1".to_vec(),
             *operator_account_1.address(),
         );
         let txn = aptos_root_account
-            .transaction()
-            .write_set(WriteSetPayload::Script {
-                script: admin_script,
-                execute_as: *validator_account.address(),
```

Comment:
hi @zekun000  this also have this execute_as. And I am not sure how to pass this in the new script function.